experiments/snippets/skuidCustom/skuidCustom.js

```diff
@@ -0,0 +1,1344 @@
+							return undefined;
+						}
+					}
+
+					if (value === undefined || value === null) {
+						value = 'NULL';
+					}
+
+					let precede = '';
+					
+					if (operator.includes('start')) {
+						operator = 'LIKE';
+						value += '%';
+					} else if (operator.includes('end')) {
+						operator = 'LIKE';
+						value = `%${value}`;
+					} else if (operator.includes('contain')) {
+						operator = 'LIKE';
+						value = `%${value}%`;
+					}
+					
+					if (operator.includes('does not')) {
+						precede = 'NOT ';
+					}
+					
+					if (value !== 'NULL' && c.encloseValueInQuotes) {
+						value = `'${value}'`;
+					}
+					
+					// LIKE NULL is invalid, switch operator to equals
+					if (operator === 'LIKE' && value === 'NULL') {
+						operator = '=';
+					}
+					
+					conditionsItems.push(`(${precede}${c.field} ${operator} ${value})`);
+				}
+			}
+
+			return conditionsItems;
+		}
+
+		function sortRows (rows, model) {
+			if (model === undefined) {
+				return rows;
+			}
+
+			const orderby = model.orderByClause;
+			if (orderby === undefined || orderby === null || orderby === '') {
+				return rows;
+			}
+
+			// turn orderby into an array of objects
+			const orderbyArr = [];
+
+			const orderbySplit = orderby.split(',');
+
+			for (let i = 0; i < orderbySplit.length; i++) {
+				let os = orderbySplit[i].trim();
+				let field;
+				let ascDesc;
+				let nullsLast = false;
+
+				// take care of nulls last condition
+				if (os.toUpperCase().endsWith(' NULLS LAST')) {
+					nullsLast = true;
+					os = os.substring(0, os.length - 11).trim();
+				}
+
+				if (os.toUpperCase().endsWith('ASC')) {
+					ascDesc = 'ASC';
+					field = os.substring(0, os.length - 3).trim();
+				} else if (os.toUpperCase().endsWith('DESC')) {
+					ascDesc = 'DESC';
+
+					field = os.substring(0, os.length - 4).trim();
+				} else {
+					field = os;
+					ascDesc = 'ASC';
+				}
+
+				if (model.isAggregate === true) {
+					field = getAggFieldNameFromId(model, field);
+				}
+
+				orderbyArr.push({ field, ascDesc, nullsLast });
+			}
+
+			rows.sort((a, b) => {
+				for (let i = 0; i < orderbyArr.length; i++) {
+					const ob = orderbyArr[i];
+					const field = ob.field;
+					const ascDesc = ob.ascDesc;
+					const nullsLast = ob.nullsLast;
+
+					const aField = Object.byString(a, field);
+					const bField = Object.byString(b, field);
+
+					if (nullsLast && aField === null && bField !== null) {
+						return 1;
+					}
+					if (nullsLast && aField !== null && bField === null) {
+						return -1;
+					}
+					
+					if (aField < bField) {
+						if (ascDesc === 'ASC') {
+							return -1;
+						} else if (ascDesc === 'DESC') {
+							return 1;
+						}
+					} else if (aField > bField) {
+						if (ascDesc === 'ASC') {
+							return 1;
+						} else if (ascDesc === 'DESC') {
+							return -1;
+						}
+					}
+				}
+
+				return 0;
+			});
+
+			return rows;
+		}
+
+		Object.byString = function (o, s) {

```

Comment:
this function is already defined above, and as described before you can use `skuid.utils.getObjectProperty(o, s)` instead.

Reply:
Removed, using skuid.utils.getObjectProperty(o, s) instead